app/javascript/controllers/edit_deposit_controller.js

```diff
@@ -29,4 +32,24 @@ export default class extends Controller {
     }
     step.classList.toggle('active', isComplete)
   }
+
+  toggleMoreTypes(event) {
+    event.preventDefault()
+
+    this.moreTypesTarget.hidden ?
+      this.showMoreTypes() :
+      this.hideMoreTypes()
+  }
+
+  showMoreTypes() {
+    this.moreTypesTarget.hidden = false
+    this.moreTypesLinkTarget.innerHTML = 'See fewer options'
+    this.moreTypesLinkTarget.classList.toggle('collapsed', false)
+  }
+
+  hideMoreTypes() {
+    this.moreTypesTarget.hidden = true
+    this.moreTypesLinkTarget.innerHTML = 'See more options'
+    this.moreTypesLinkTarget.classList.toggle('collapsed', true)
+  }
```

Comment:
Copypasta from the work_types_controller.js. That controller contains other behavior we don't want in this context, so instead of a gnarly refactoring to reduce duplication, just do the simple thing: copy and paste the code.